### ISSUE TITLE
(Nice to have) Allow complete customization of string formatting in the preferences page

### ISSUE THREAD

**Issue by sharky98** [kimai/kimai#4430]
### Describe the problem

The formatting I like to use if often a mix of different "languages" formatting. My own ideal view for my picky brain is the following, which no locales cater to (obviously! 😅).

- ISO date `2035-11-25`;
- 24-hours clock with a colon `15:34`;
- A dot decimal separator `12.81`;
- A space thousand separator `2 794.83`;
- Dollar sign after the number, with a space `2 794.83 $`.

### Describe the solution you'd like

Added on top of proposed solution in #4429, having a `<select/>` field for each kind of value to select a specific format, similar to what the [Windows Region and Language dialog](https://techcommunity.microsoft.com/t5/ask-the-performance-team/how-to-change-regional-settings-for-all-users-on-a-computer/ba-p/375086) is doing.

### Describe alternatives you've considered

An less intensive alternative (but not fully answering to the problem) would be to split the interface language and format language so that a user could have a bit more control (like me having the `français (Canada)` interface language but using `svenska (Finland)` for string format as being the closest to my picky brain expectation!).

**Comment by kevinpapst** [comment]
Yes, something like that is planned. I doubt that I will offer every single field.
But splitting locale for translations and formats, seems to best the best trade-off for now, between required work and potential win.

**Comment by sierja** [comment]
This probably supersedes #3934
